test/usecases/kinetic/X2Y.mod

```diff
@@ -0,0 +1,30 @@
+NEURON {
+    SUFFIX pump
+    RANGE X, Y
+    NONSPECIFIC_CURRENT il
+}
+
+STATE {
+    X
+    Y
+}
+
+ASSIGNED {
+    il
+    i
+}
+
+BREAKPOINT {
+    SOLVE state METHOD sparse
+    il = i
+}
+
+INITIAL {
+    X = 0
+    Y = 1
+}
+
+KINETIC state {
+    ~ X <-> Y (0.4, 0.5)
+    i = (f_flux-b_flux)
```

Comment:
I don't see `b/f_flux` anywhere in the generated code (or something close to those var names) so I'm curious, how does this work? Is this an internal NEURON var?

Reply:
In the line above there's material flowing from the left to the right (and vice-versa), the amount is available via `{f,b}_flux`.

Reply:
Ah, found some docs online: https://www.neuron.yale.edu/neuron/static/docs/refman/kinetic.html
Unrelated, I feel like this should be part of the RTD docs because the [search](https://nrn.readthedocs.io/en/latest/search.html?q=f_flux&check_keywords=yes&area=default) reveals nothing useful...